apps/vaults/components/details/tabs/VaultDetailsAbout.tsx

```diff
@@ -11,60 +11,68 @@ import type {TGraphForVaultEarningsProps} from '@vaults/components/graphs/GraphF
+					<p className={'mt-4 text-neutral-600'} dangerouslySetInnerHTML={{__html: getVaultDescription()}} />
 				</div>
 				<div>
 					<b className={'text-neutral-900'}>{'APY'}</b>
 					<div className={'mt-4 grid grid-cols-1 gap-x-12 md:grid-cols-2'}>
 						<div className={'space-y-2'}>
-							<div className={'flex flex-row items-center justify-between'}>
-								<p className={'text-sm text-neutral-500'}>{'Weekly APY'}</p>
-								<p className={'font-number text-sm text-neutral-900'} suppressHydrationWarning>
-									{formatPercent((currentVault?.apy?.points?.week_ago || 0) * 100)}
-								</p>
-							</div>
-							<div className={'flex flex-row items-center justify-between'}>
-								<p className={'text-sm text-neutral-500'}>{'Monthly APY'}</p>
-								<p className={'font-number text-sm text-neutral-900'} suppressHydrationWarning>
-									{formatPercent((currentVault?.apy?.points?.month_ago || 0) * 100)}
-								</p>
-							</div>
-							<div className={'flex flex-row items-center justify-between'}>
-								<p className={'text-sm text-neutral-500'}>{'Inception APY'}</p>
-								<p className={'font-number text-sm text-neutral-900'} suppressHydrationWarning>
-									{formatPercent((currentVault?.apy?.points?.inception || 0) * 100)}
-								</p>
-							</div>
+							<APYLineItem label={'Weekly APY'} value={apy.points.week_ago} />
+							<APYLineItem label={'Monthly APY'} value={apy.points.month_ago} />
+							<APYLineItem label={'Inception APY'} value={apy.points.inception} />
 						</div>
-						<div className={'space-y-2'}>
-							<div className={'flex flex-row items-center justify-between'}>
-								<p className={'text-sm text-neutral-500'}>{'Gross APR'}</p>
-								<p className={'font-number text-sm text-neutral-900'} suppressHydrationWarning>
-									{formatPercent((currentVault?.apy?.gross_apr || 0) * 100)}
-								</p>
-							</div>
-							<div className={'flex flex-row items-center justify-between'}>
-								<p className={'text-sm text-neutral-500'}>{'Net APY'}</p>
-								<p className={'font-number text-sm text-neutral-900'} suppressHydrationWarning>
-									{formatPercent(((currentVault?.apy?.net_apy || 0) + (currentVault?.apy?.staking_rewards_apr || 0)) * 100, 2, 2, 500)}
-								</p>
-							</div>
-							{currentVault.apy?.staking_rewards_apr > 0 && (
-								<div className={'flex flex-row items-center justify-between'}>
-									<p className={'text-sm text-neutral-500'}>{'Reward APR'}</p>
-									<p className={'font-number text-sm text-neutral-900'}>
-										{formatPercent((currentVault?.apy?.staking_rewards_apr || 0) * 100, 2, 2, 500)}
-									</p>
-								</div>
-							)}
+						<div className={'mt-2 space-y-2 md:mt-0'}>
```

Comment:
The real fix, add margin top when in mobile view, as this `div` falls underneath the previous one and didn't have margin